documentation/src/main/asciidoc/topics/proc_configuring_protobuf_encoding.adoc

```diff
@@ -0,0 +1,33 @@
+[id='configuring_protobuf_encoding-{context}']
+= Using Protobuf Encoding for Data
+Storing data in the cache as Protobuf encoded entries provides a platform
+independent configuration that enables you to perform cache operations from any
+client.
+
+.Procedure
+
+. Specify `application/x-protostream` as the MediaType for keys and values as follows:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+include::config_examples/encoding_protostream.xml[]
+----
++
+[NOTE]
+====
+When you configure indexing, {brandname} automatically stores keys and values
+with the `application/x-protostream` media type.
+====
++
+. Configure your clients.
+
+For Hot Rod clients, you must register Protocol Buffers schema definitions to
+describe the entities and marshallers that the clients use.
+
+{brandname} automatically converts between `application/x-protostream` and
+`application/json` so that REST clients can read and write JSON formatted data.
```

Comment:
Suggestion:
```
{brandname} supports converions between `application/x-protostream` and
`application/json` so that REST clients can read and write JSON formatted data.
```